### ISSUE TITLE
yarn build bumps errors

### ISSUE THREAD

**Issue by app-generator** [app-generator/nextjs-material-kit#1]
The execution of `yarn build` bumps errors. 

> Environment

- Node `v16.15.0`
- yarn `1.22.18`

```log
yarn ; yarn build
yarn install v1.22.18
[1/4] Resolving packages...
success Already up-to-date.
Done in 0.19s.
yarn run v1.22.18
$ next build
info  - Linting and checking validity of types  
info  - Creating an optimized production build  
Failed to compile.

./pages-sections/LandingPage-Sections/ProductSection.tsx
Module not found: Can't resolve '../../components/Grid/GridContainer.js' in 'D:\work\repo-free\nextjs-material-kit\pages-sections\LandingPage-Sections'

Import trace for requested module:
./pages/landing.tsx

./pages-sections/LandingPage-Sections/ProductSection.tsx
Module not found: Can't resolve '../../components/Grid/GridItem.js' in 'D:\work\repo-free\nextjs-material-kit\pages-sections\LandingPage-Sections'

Import trace for requested module:
./pages/landing.tsx

./pages-sections/LandingPage-Sections/ProductSection.tsx
Module not found: Can't resolve '../../components/InfoArea/InfoArea.js' in 'D:\work\repo-free\nextjs-material-kit\pages-sections\LandingPage-Sections'

Import trace for requested module:
./pages/landing.tsx

./pages-sections/LandingPage-Sections/TeamSection.tsx
Module not found: Can't resolve '../../components/Grid/GridContainer.js' in 'D:\work\repo-free\nextjs-material-kit\pages-sections\LandingPage-Sections'

Import trace for requested module:
./pages/landing.tsx

./pages-sections/LandingPage-Sections/TeamSection.tsx
Module not found: Can't resolve '../../components/Grid/GridItem.js' in 'D:\work\repo-free\nextjs-material-kit\pages-sections\LandingPage-Sections'

Import trace for requested module:
./pages/landing.tsx


> Build failed because of webpack errors
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by rohitrehan** [comment]
This is fixed in commit https://github.com/app-generator/nextjs-material-kit/commit/467d995b17973b29d8efda1bf7bafdff9660cd1e

**Comment by app-generator** [comment]
Fixed in tag `v1.2.2`
